install.sh

```diff
@@ -41,8 +41,32 @@ if [[ -z "${apiKey}" ]]; then
 fi
 
 # Add the Aspen Mesh Helm Repository
-helm repo add aspenmesh https://aspenmesh.github.io/aspenmesh-charts/
-helm repo update
+aspenMeshHelmChartRepo="https://aspenmesh.github.io/aspenmesh-charts/"
+helmAddErr=$(helm repo add aspenmesh ${aspenMeshHelmChartRepo} 2>&1 >/dev/null || true)
+helmUpdateErr=$(helm repo update 2>&1 >/dev/null || true)
+if [[ -n "${helmAddErr}" || -n "${helmUpdateErr}" ]]; then
+  echo ""
+  echo "***Failed to add Aspen Mesh Chart helm repository ${aspenMeshHelmChartRepo}***"
+  echo "Please send the following error messages to hello@aspenmesh.io:"
+  echo "${helmAddErr}"
+  echo "${helmUpdateErr}"
+  echo ""
```

Comment:
A couple notes:
- it might make it a little easier to read if we collapsed into one `echo` statement?
- This is more a question for Rick and Joanna, but maybe we want to make the output look more like an email template? like:

```suggestion
  echo ""
  echo "
***Failed to add Aspen Mesh Chart helm repository ${aspenMeshHelmChartRepo}***
Please send the following message to hello@aspenmesh.io:

I had issues accessing the Aspen Mesh collector helm repository:

${helmAddErr}
${helmUpdateErr}

"
```